pulumi-deck/rollup.config.js

```diff
@@ -0,0 +1,13 @@
+const basePluginConfig = require('@spinnaker/pluginsdk/pluginconfig/rollup.config');
+import injectProcessEnv from 'rollup-plugin-inject-process-env';
+
+export default {
+  ...basePluginConfig,
+  plugins: [
+    ...basePluginConfig.plugins,
+    injectProcessEnv({
+      NODE_ENV: 'production',
```

Comment:
@danielpeach Although this isn't really needed, I defined this because I was getting errors like `process is undefined...` anytime I used a React-based component library in this plugin. I finally didn't end up using any such libraries and found replacements I could use from `@spinnaker/core` but left this intact. Let me know if it is recommended to remove it.

Reply:
Not familiar with that error... that's pretty weird. Is it really any component library?

Reply:
Yeah it was for any component library.

Reply:
@praneetloke this was fixed in the latest version of `@spinnaker/pluginsdk` (0.0.26).